packages/patternfly-4/react-table/src/components/Table/base/body-row.tsx

```diff
@@ -1,29 +1,40 @@
 /**
- * body-row.js
+ * body-row.tsx
  *
  * Forked from reactabular-table version 8.14.0
  * https://github.com/reactabular/reactabular/tree/v8.14.0/packages/reactabular-table/src
- * */
-
+ */
 import { isEqual, isFunction } from 'lodash';
-import React from 'react';
-import columnsAreEqual from './columns-are-equal';
-import evaluateFormatters from './evaluate-formatters';
-import evaluateTransforms from './evaluate-transforms';
-import mergeProps from './merge-props';
-import { tableBodyRowDefaults, tableBodyRowTypes } from './types';
+import * as React from 'react';
+import { columnsAreEqual } from './columns-are-equal';
+import { evaluateFormatters } from './evaluate-formatters';
+import { evaluateTransforms } from './evaluate-transforms';
+import { mergeProps } from './merge-props';
+import { createElementType, formatterValueType, ColumnType, RowType, RenderersTypes } from './types';
+
+export interface BodyRowProps {
+  columns: RenderersTypes['columns'];
+  renderers: RenderersTypes['renderers']['body'];
+  onRow?: Function;
+  rowIndex: number;
+  rowData: RowType;
+  rowKey: string;
+}
 
-class BodyRow extends React.Component {
-  shouldComponentUpdate(nextProps) {
-    // eslint-disable-line no-unused-vars
+export class BodyRow extends React.Component<BodyRowProps, {}> {
+  static defaultProps = {
+    onRow: (...args:any) => Object 
+  };
+
+  shouldComponentUpdate(nextProps: BodyRowProps){
     const previousProps = this.props;
```

Comment:
We can probably remove this `previousProps`, right? It was never used anyway, i'm not sure why we had the `// eslint-disable-line no-unused-vars`

Reply:
yes... that is old code in Reactabular ;) - good call... updated to destructure those from `this.props` instead:

```
const {columns, rowData} = this.props;
```